src/Symfony/Component/Cache/Tests/Adapter/AbstractRedisAdapterTest.php

```diff
@@ -34,9 +34,10 @@ public static function setUpBeforeClass(): void
         if (!\extension_loaded('redis')) {
             self::markTestSkipped('Extension redis required.');
         }
-        if (!@((new \Redis())->connect(getenv('REDIS_HOST')))) {
-            $e = error_get_last();
-            self::markTestSkipped($e['message']);
+        try {
+            (new \Redis())->connect(getenv('REDIS_HOST'));
```

Comment:
We are not sure that the RedisExtension is loaded here. This is also used by Predis.

Reply:
There's a check for the extension few lines above.